lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -779,11 +780,40 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   end
 
   defp handle_media_event(%{type: :sdp_offer, data: data}, ctx, state) do
-    state = Map.put(state, :track_id_to_metadata, data.track_id_to_track_metadata)
-    msg = {:signal, {:sdp_offer, data.sdp_offer.sdp, data.mid_to_track_id}}
+    map_track_info = fn map ->
+      data.track_id_to_track_info
+      |> Enum.map(map)
+      |> Map.new()
+    end
+
+    track_id_to_track_metadata =
+      map_track_info.(fn {id, track_info} -> {id, track_info.track_metadata} end)
+
+    track_id_to_bandwidth =
+      map_track_info.(fn {id, track_info} -> {id, track_info.max_bandwidth} end)
```

Comment:
I add `track_id_to_bandwidth` map to state, it's worth mentioning that its keys are `rid` string and I was thinking that maybe I should convert it to map with atom keys (`:high` etc.) at the beginning and not when it is passed to `TrackSender`.